### ISSUE TITLE
Place ternary operator and colon tokens at begin of operation and not at end

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1513]
Following code should issue a warning:

```c#
var result = xyz == 42 ?
"someText" :
"someOtherText";
```

It should be changed into
```c#
var result = xyz == 42
? "someText"
: "someOtherText";
```
